scene/gui/tab_container.h

```diff
@@ -47,6 +47,8 @@ class TabContainer : public Container {
 	bool theme_changing = false;
 	Vector<Control *> children_removing;
 	bool drag_to_rearrange_enabled = false;
+	bool setting_up = false;
+	int setup_current_tab = 0;
```

Comment:
```suggestion
	int setup_current_tab = -1;
```
Instead of `setting_up` you can check if `setup_current_tab >= 0`.